### DISCUSSION TITLE
Seo article schema for news portal website.

### DISCUSSION THREAD

**Discussion by akash-mahmud** [garmeeh/next-seo#1119] [Q&A]
I have used Seo schema package for on of my package and it is pretty good. My project is basically a news portal website. For my article details page, I want to use structured content or SEO schema. How can I achieve this functionality using next-seo? 
Any suggestions will be appreciated. 
Thanks in advance.

**Comment by zuridantethea** [comment]
Great discussion, everyone! I've been looking into different SEO tools lately, and I noticed that [Pearl Lemon Leads reviews](https://pearllemonleads.com/testimonials-reviews/) really highlight how important good SEO is for generating leads. It’s amazing how the right strategies can make such a big difference in getting noticed and attracting clients
